app/js/app/directives/content/StringMatchQuestion.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2017 James Sharkey
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define(["app/honest/responsive_video"], function(rv) {
```

Comment:
`app/honest/responsive_video`?

Reply:
I don't know, I copied it from `NumericQuestion.js`, and it appears in all the question type JS files, so I'm going to assume it's either out-of-date and a relic, or needs to be there :P

Reply:
@daviesian may know!